app/src/main/java/com/kamron/pogoiv/activities/OcrCalibrationResultActivity.java

```diff
@@ -105,7 +105,11 @@ public static void startCalibration(@NonNull Context context,
             sCalibrationImage = bitmap;
         } else {
             // Make a mutable copy of the bitmap so we can draw on it with a Canvas
-            sCalibrationImage = bitmap.copy(sCalibrationImage.getConfig() ,true);
+            sCalibrationImage = bitmap.copy(Bitmap.Config.ARGB_8888 ,true);
+            if (sCalibrationImage == null) {
```

Comment:
This check is useless since the copy can't return a null bitmap.
Please remove this added lines since they're not helpful.

Reply:
It could be NULL according to the reference:
```
If the conversion is not supported, or the allocator fails, then this returns NULL
```
but yea, I think it's a pretty rare case and maybe just doesn't worth bothering at all